### PR TITLE
docs: edits to remove jargon

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/service-worker-config.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ Unless otherwise noted, patterns use a limited glob format:
...
+* The `!` prefix marks the pattern as being negative, meaning that only files that don't match the pattern are included.
 
 Example patterns:
 
 * `/**/*.html` specifies all HTML files.
 * `/*.html` specifies only HTML files in the root.
 * `!/**/*.map` exclude all sourcemaps.
 
-Each section of the configuration file is described below.
+Each section of the configuration file is described in the following section.
```
Comment (gkalpak) [inline aio/content/guide/service-worker-config.md:36]: This sounds a little confusing to me, because (a) we use the word "section" to refer to two different things in the same sentence and (b) there is not _one_ following section that describes the configuration file properties, but one section per property. So, maybe change it to something like:

```suggestion
The following sections describe each property of the configuration file.
```

File: aio/content/guide/service-worker-config.md [hunk 2]
```diff
@@ -256,11 +256,11 @@ This optional section enables you to specify a custom list of URLs that will be
 
 ### Handling navigation requests
 
-The ServiceWorker will redirect navigation requests that don't match any `asset` or `data` group to the specified [index file](#index-file). A request is considered to be a navigation request if:
+The ServiceWorker redirectd navigation requests that don't match any `asset` or `data` group to the specified [index file](#index-file). A request is considered to be a navigation request if:
```
Comment (gkalpak) [inline aio/content/guide/service-worker-config.md:259]:
```suggestion
The ServiceWorker redirects navigation requests that don't match any `asset` or `data` group to the specified [index file](#index-file). A request is considered to be a navigation request if:
```

File: aio/content/guide/service-worker-config.md [hunk 3]
```diff
@@ -256,11 +256,11 @@ This optional section enables you to specify a custom list of URLs that will be
...
 
 1. Its [mode](https://developer.mozilla.org/en-US/docs/Web/API/Request/mode) is `navigation`.
 2. It accepts a `text/html` response (as determined by the value of the `Accept` header).
-3. Its URL matches certain criteria (see below).
+3. Its URL matches certain criteria (see the following).
```
Comment (gkalpak) [inline aio/content/guide/service-worker-config.md:263]: Is that correct? (non-native speaker question 😅)

File: aio/content/guide/service-worker-devops.md [hunk 4]
```diff
@@ -356,12 +356,12 @@ entirely from Service Worker. The old worker (registered at `example.com`)
  get redirected to the new location `www.example.com` and create the error
 `The script resource is behind a redirect, which is disallowed`.
 
-To remedy this, you may need to kill the old worker using one of the above
+To remedy this, you might need to stop the old worker using one of the above
```
Comment (gkalpak) [inline aio/content/guide/service-worker-devops.md:359]: In the context of ServiceWorkers, it might be more appropriate to say "get rid of" or "deactivate" or "unregister".

File: aio/content/guide/service-worker-intro.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Angular service worker introduction
 
-Service workers augment the traditional web deployment model and empower applications to deliver a user experience with the reliability and performance on par with natively-installed code. Adding a service worker to an Angular application is one of the steps for turning an application into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) (also known as a PWA).
+Service workers augment the traditional web deployment model and empower applications to deliver a user experience with the reliability and performance on par with built-in installed code. Adding a service worker to an Angular application is one of the steps for turning an application into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) (also known as a PWA).
```
Comment (gkalpak) [inline aio/content/guide/service-worker-intro.md:3]: Why this change? "Native" sounds more appropriate/accurate than "built-in" here :confused:

File: aio/content/guide/service-worker-notifications.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ The Angular service worker enables the display of push notifications and the han
 <div class="alert is-helpful">
 
   When using the Angular service worker, push notification interactions are handled using the `SwPush` service.
-  To learn more about the native APIs involved see [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) and [Using the Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API).
+  To learn more about the built-in APIs involved see [Push API](https://developer.mozilla.org/en-US/docs/Web/API/Push_API) and [Using the Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API/Using_the_Notifications_API).
```
Comment (gkalpak) [inline aio/content/guide/service-worker-notifications.md:10]: Again, "built-in" is not very clear/idiomatic here.
Maybe use "browser APIs" instead? (Reference: [MDN][1])

[1]: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Introduction#apis_in_client-side_javascript

**Review by aikithoughts — COMMENTED** [review]
Looking good; should address gkalpak's comments though. Then we can move foward!

File: aio/content/guide/service-worker-config.md (hunk 1 above)
Comment (TeriGlover) [reply aio/content/guide/service-worker-config.md:36]: Done

File: aio/content/guide/service-worker-config.md (hunk 2 above)
Comment (TeriGlover) [reply aio/content/guide/service-worker-config.md:259]: done

File: aio/content/guide/service-worker-config.md (hunk 3 above)
Comment (TeriGlover) [reply aio/content/guide/service-worker-config.md:263]: Yes

File: aio/content/guide/service-worker-devops.md (hunk 4 above)
Comment (TeriGlover) [reply aio/content/guide/service-worker-devops.md:359]: Used "deactivate"

File: aio/content/guide/service-worker-notifications.md (hunk 6 above)
Comment (TeriGlover) [reply aio/content/guide/service-worker-notifications.md:10]: Done

File: aio/content/guide/service-worker-intro.md (hunk 5 above)
Comment (TeriGlover) [reply aio/content/guide/service-worker-intro.md:3]: Changed to: Service workers augment the traditional web deployment model and empower applications to deliver a user experience with the reliability and performance on par with code that is written to run on your operating system and hardware.

File: aio/content/guide/service-worker-devops.md (hunk 4 above)
Comment (aikidave) [reply aio/content/guide/service-worker-devops.md:363]: Hi @gkalpak ! For reference, here's the entry in the Google Developers Style Guide: https://developers.google.com/style/word-list?hl=en#letter-b. I'm afraid it doesn't explain *why* `following` is better, but we do try to comply with the style guide as much as possible!

File: aio/content/guide/service-worker-config.md (hunk 3 above)
Comment (aikidave) [reply aio/content/guide/service-worker-config.md:272]: Hi @gkalpak ! For reference, here's the entry in the Google Developers Style Guide: https://developers.google.com/style/word-list?hl=en#letter-b. I'm afraid it doesn't explain why following is better, but we do try to comply with the style guide as much as possible!

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvals

**Review by alxhub — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]
LGTM. Thanks Teri!

**Comment by IgorMinar** [comment]
merge-assistance note: for some reason I can't remove the pending reviewers, which causes the CI status to be still "pending". This PR has sufficient reviews already, please go ahead and merge.
